lib/core/wireprotocol/command.js

```diff
@@ -9,6 +9,10 @@ const databaseNamespace = require('./shared').databaseNamespace;
 const isTransactionCommand = require('../transactions').isTransactionCommand;
 const applySession = require('../sessions').applySession;
 
+function isClientEncryptionEnabled(server) {
+  return server && server.s && server.s.options && server.s.options.autoEncrypter;
```

Comment:
This could be follow-on work, but it would be nice to have `server.autoEncrypter` similar to `collection.readConcern` so we don't have to do this long check here.

Reply:
Agreed, created [NODE-2013](https://jira.mongodb.org/browse/NODE-2013) to track this.